getting-started/installation-windows.rst

```diff
@@ -90,18 +97,31 @@ Install EPICS
 
 There should be lots of warnings, but no error. 
 
-Test EPICS
-----------
+Check EPICS Installation
+------------------------
 
 Run ``softIoc`` and, if everything is ok, you should see an EPICS prompt.
 
 ::
 
-    softIoc
-    epics>
+    $ /home/Hitesh/base-7.0.3.1/bin/windows-x64-mingw/softIoc
+    $ epics>
 
 You can exit with ctrl-c or by typing exit.
 
 Voilà.
 
 Ok, now you know that EPICS is installed correctly.
+
+Test EPICS in Windows
+---------------------
+
+Exit or minimise Msys2 environment. Open windows command prompt.
+
+::
+
+     cd c:\msys64\home\Hitesh\base-7.0.3.1\bin\windows-x64-mingw
```

Comment:
Since you added the bash prompt earlier, shouldn't this example show the cmd.exe prompts to match?

Same comment as above regarding the path to the installation, but I'm not sure that `%HOME%` would be set, I suspect that's something that MSYS may be setting when you run bash, which isn't happening here. That's why using `C:\epics` might be better.

Reply:
I tried to use bash prompt but it is symbol ">", which can get confused with when IOC starts and "epics>" appears. So, I removed it. 

For Path, I would remove name and put 'user' into it. I think putting actual path will help users, as switching between msys and windows bash will require different types of path. So, I put whole paths when using windows bash and relative path in while using Msys. 

I am trying to prepare documentation such that, If any person follows instruction as it is and copies the commands, it should work without any modification.

If I use path like c:\epics, Msys would not have access to that location.

Reply:
Relavent changes made and committed.